test/integration/tests/veBalHelper_gauge.ts

```diff
@@ -102,41 +103,4 @@ describe('e2e-veBalHelper-gauge-management', function () {
     ).sub(stakeBeforeBalance);
     expect(stakeBalanceDiff).to.equal(0);
   });
-
-  it('should be able to voteForGaugeWeight() to vote for gauge weights whilst a lock is active ', async () => {
```

Comment:
Removed test as Aave are actively voting, and even when fast forward time am getting errors around using too much vote power. The veBAL tests as a whole are only being left in as a sanity check that Aave will still be able to operate the veBAL, fine imo to remove this one